ytrello.rb

```diff
@@ -60,23 +61,28 @@
 ENV_TRELLO_KEY = "TRELLO_DEVELOPER_PUBLIC_KEY".freeze
 ENV_TRELLO_TOKEN = "TRELLO_MEMBER_TOKEN".freeze
 
-def check_trello_credentials
-  return if ENV[ENV_TRELLO_KEY] && ENV[ENV_TRELLO_TOKEN]
-  $stderr.puts "Error: Pass the Trello credentials via #{ENV_TRELLO_KEY}" \
-  " and\n#{ENV_TRELLO_TOKEN} environment variables."
-  exit 1
-end
-
 # set the SUSE Bugzilla connection
 def setup_bicho
   Bicho.client = Bicho::Client.new(BUGZILLA_URL)
 end
 
 # set the Trello credentials
 def setup_trello
+  fn = "#{ENV["HOME"]}/.config/trello-creds.yml"
+  text = begin; File.read(fn); rescue Errno::ENOENT; ""; end
```

Comment:
What about 
```ruby
text = File.exist?(fn) ? File.read(fn) : ""
```

IMHO better readable without the exception handling.

Reply:
@lslezak you're absolutely right, thanks!